kollider-price/src/price_feed/mod.rs

```diff
@@ -0,0 +1,43 @@
+use futures::{SinkExt, Stream, StreamExt};
+
+use config::KolliderPriceFeedConfig;
+use error::KolliderPriceFeedError;
+pub use tick::*;
+use tokio_tungstenite::{connect_async, tungstenite::Message};
+
+pub mod config;
+pub mod error;
+mod tick;
+
+pub async fn subscribe_price_feed(
+    config: KolliderPriceFeedConfig,
+) -> Result<std::pin::Pin<Box<dyn Stream<Item = KolliderPriceTicker> + Send>>, KolliderPriceFeedError>
+{
+    let (ws_stream, _) = connect_async(config.url).await?;
+
+    let (mut sender, receiver) = ws_stream.split();
+
+    let subscribe_args = serde_json::json!({
+        "type": "subscribe",
+        "symbols": ["BTCUSD.PERP"],
+        "channels": ["ticker"]
+    })
+    .to_string();
+    let item = Message::Text(subscribe_args);
+
+    sender.send(item).await.unwrap();
```

Comment:
With `.unwrap()` the program will panic if there's an error. We generally avoid using it in production code, preferring to propagate errors.

```
sender.send(item).await?;
```